www/src/components/docs/breadCrumbs.tsx

```diff
@@ -0,0 +1,54 @@
+export default function BreadCrumbs() {
+  const breadcrumbs = window.location.href
+    .split("/")
+    .slice(window.location.href.split("/").length > 5 ? -2 : -1)
+    .map((crumb) => {
+      const href = window.location.href
+        .split("/")
+        .slice(0, window.location.href.split("/").indexOf(crumb) + 1)
+        .join("/");
+      return {
+        href,
+        text: crumb,
```

Comment:
I wonder if we shouldn't grab this text from the sidebar instead? We could make some util that finds the link and maps it to the correct text. (This would also make it work for translations)

Right now there are a few weird ones like
<img width="227" alt="CleanShot 2022-11-28 at 14 54 46@2x" src="https://user-images.githubusercontent.com/51714798/204295196-edcb82bc-281d-4eab-9ff4-6ed354ee9e74.png">
<img width="266" alt="CleanShot 2022-11-28 at 14 54 39@2x" src="https://user-images.githubusercontent.com/51714798/204295228-d592903a-104a-44ef-b2eb-da1b00bdc207.png">
<img width="280" alt="CleanShot 2022-11-28 at 14 54 34@2x" src="https://user-images.githubusercontent.com/51714798/204295247-e380cc46-10fb-46c2-9425-cc3ecadefb14.png">

Reply:
done via https://github.com/t3-oss/create-t3-app/commit/330f419581e5b3f6cdf418095a797f129056c112